src/main/java/frc/robot/Robot.java

```diff
@@ -108,6 +132,74 @@ public void autonomousInit()
 
     //new TargetingCommandGroup(1, true).start();
     //new TargetingCommandGroup(2, true).start();
+
+    //Sets the flags for the hatch holder and ball intake.
+    if (RobotMap.gamePieceChooser.getSelected().equals("hatch "))
+    {
+      RobotMap.ballInIntake = false;
+      RobotMap.hatchHolderHasHatch = true;
+    }
+
+    RobotMap.autonomousPath = RobotMap.startingPositionChooser.getSelected();
+
+    //Because the middle portion on the rocket is Cargo only, it has a different path name.
+    if (RobotMap.gamePieceChooser.getSelected().equals("Cargo ") 
```

Comment:
If this block is why the trailing space is in the gamePieceChooser settings, you may want to consider just using String.format to construct the Strings instead.

Reply:
See my above comment